src/components/reusable/radioButton/radioButtonGroup.ts

```diff
@@ -90,12 +90,24 @@ export class RadioButtonGroup extends FormMixin(LitElement) {
           : null}
 
         <div class="${this.horizontal ? 'horizontal' : ''}">
-          <slot></slot>
+          <slot @slotchange=${this._handleSlotChange}></slot>
         </div>
       </fieldset>
     `;
   }
 
+  private _handleSlotChange() {
+    this._updateChildren();
+    this.requestUpdate();
+  }
+
+  private _updateChildren() {
```

Comment:
You've got the right idea now, but need to take it a bit further. All of those props within the `updated()` function that cause updates to child radio buttons should also probably go in here. Then you can just call this function in both places.

Reply:
Right you'd have to call `updateChildren` in both `slotchange` and `updated()`

Reply:
Pushed the changes